legion/legion/http.py

```diff
@@ -103,9 +106,14 @@ def decorated_function(*args, **kwargs):
         except legion.utils.EdiHTTPException as edi_http_exception:
             response = {'error': True, 'message': edi_http_exception.message}
             code = edi_http_exception.http_code
+            LOGGER.exception('Internal error during processing request for {}: {}'
```

Comment:
Why do we ignore the error here?

Reply:
Why ignore? We save exception to log and return to client

Reply:
Have come to the conclusion that I have to replace that with normal `Exception system` and I would not mix application-level and protocol-level codes of errors.